src/js/utils.js

```diff
@@ -16,6 +16,27 @@ exports.is_sample_genetically_altered = function is_sample_genetically_altered(d
   || datum.protein !== undefined;
 };
 
+exports.makeD3SVGElement = function(tag) {
+  return d3.select(document.createElementNS('http://www.w3.org/2000/svg', tag));
+};
+
+exports.stableSort = function(arr, cmp) {
```

Comment:
Here's my convoluted counter example. The whole `max_length` business is something extra to make it make a bit more sense, but I think that generally the sort that you want on a list of strings of varying lengths depends on the problem.

```js
var local = 'de';
function sortByUsingLocalCompare(list) {
    var sorted = _.sortBy(list, function(x) { return x.length; });
    var max_length = sorted[sorted.length-1].length;

    var zero = '';
    for (var i=0; i<max_length; i++)
        zero+='a';

    return _.sortBy(list, function(x) {
        return x.localeCompare(zero, local);
    });
}
```